src/Monad/Writer/functions.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Widmogrod\Monad\Writer;
+
+use Widmogrod\Monad as M;
+
+const pure = 'Widmogrod\Monad\Writer\pure';
+
+/**
+ * pure :: Applicative Just f => a -> f a
+ *
+ * @param mixed $f
+ * @return M\Writer
+ */
+function pure($f)
+{
+    return M\Writer::of($f);
+}
+
+const value = 'Widmogrod\Monad\Writer\value';
+
+/**
+ * value :: Writer w m => a -> m a
+ *
+ * Replace the value inside the monad
+ *
+ * @param mixed $value
+ * @return callable
+ */
+function value($value)
+{
+    return function() use ($value) {
+        return M\Writer::of($value);
```

Comment:
This function may be valid in state context I'm not sure if this is still true for Writer